lib/ui/window/platform_configuration.cc

```diff
@@ -85,46 +82,79 @@ void PlatformConfiguration::DidCreateIsolate() {
   report_timings_.Set(tonic::DartState::Current(),
                       Dart_GetField(library, tonic::ToDart("_reportTimings")));
 
-  library_.Set(tonic::DartState::Current(),
-               Dart_LookupLibrary(tonic::ToDart("dart:ui")));
+  // TODO(loicsharma): Can we remove this?
```

Comment:
Both places in this file can be removed. They're already not in the main.

Reply:
Removed